### ISSUE TITLE
Allow code tags execution only in specific contexts/windows

### ISSUE THREAD

**Issue by disconnect3d** [hackvertor/hackvertor#125]
Hi,

Thanks for a cool plugin :)

I think it may be a good idea to have an option to allow code tags execution only in specific contexts, like, e.g., I want it to be executed only in repeater window.

I have no idea if this can be implemented since I haven't looked into Burp APIs for ages, but I believe this would be good to have.

**Comment by disconnect3d** [comment]
I would also like to point out that the hash/key that is required to execute code tags is very unclear at first. I only found out about it by googling and finding this tweet: https://x.com/garethheyes/status/1186726604816429056

Either the Hackvertor input/output window could detect that the hash is missing/wrong in such code tags and inform the user about it (at least in simple examples, heh?), or, there could be some documentation or simply examples at the repository explaining this thing.

An alternative could be a popup informing the user that this is required when they click or use the code tag for the first time or something. The popup could also show the currently generated key/hash too.

Random thought: if we would limit code tags execution to just repeater, then the key/hash requirement could in theory be omitted, but oh well...

**Comment by disconnect3d** [comment]
Ugh, I just noticed there is 'Allow tags in ...' in the menu, so the main issue I asked for is actually there.

However, the other issue/point about the code tag hash/key is still valid.

**Comment by hackvertor** [comment]
Yeah agree I should have some docs on this and there should be UI to add the key to an existing tag. I'll do both but not sure when
